prompting/llms/vllm_llm.py

```diff
@@ -189,8 +189,8 @@ def _make_prompt(self, messages: list[dict[str, str]]) -> str:
     def _forward(self, messages: list[dict[str, str]]):
         # make composed prompt from messages
         composed_prompt = self._make_prompt(messages)
-        response = self.llm_pipeline(composed_prompt, **self.model_kwargs)
+        response: RequestOutput = self.llm.generate(composed_prompt, SamplingParams(**self.model_kwargs))[0]
 
-        logger.info(f"{self.__class__.__name__} generated the following output:\n{response}")
+        logger.info(f"{self.__class__.__name__} generated the following output:\n{response.outputs[0].text}")
 
-        return response
+        return response.outputs[0].text
```

Comment:
If we are dropping batch support, we should change argument from `messages: list[...]` to `message: str`

Reply:
I think I'm not understanding what you mean. _make_prompt is typed to return a string, not a list of strings?

Reply:
if I understand this correctly, we are always taking the first batch output?

Reply:
We should revisit this and clean it up at some point. Right now I just made it so it continues working with the code, but this is 100% an ugly implementation